File: src/server/src/main/java/com/project/utilities/Time.java
package com.project.utilities;

import java.time.ZoneId;
import java.time.ZonedDateTime;

public class Time {
    private Time() {
    }

    public static ZonedDateTime utcNow() {
        return ZonedDateTime.now(ZoneId.of("UTC"));
    }
}

File: src/server/src/main/kotlin/com/project/infrastructure/utilities/Time.kt
package com.project.infrastructure.utilities

import java.time.Instant
import java.time.ZoneId

object Time {
    @JvmStatic
    fun utcNow(): Instant {
        return Instant.now()
    }

    @JvmStatic
    val utcZone = ZoneId.of("UTC")
}
